src/sidebar/components/Annotation/test/AnnotationEditor-test.js

```diff
@@ -36,6 +37,12 @@ describe('AnnotationEditor', () => {
   }
 
   beforeEach(() => {
+    fakeAnnotationMetadata = {
+      annotationRole: sinon.stub().returns('Annotation'),
+      isReply: sinon.stub().returns(false),
+      isSaved: sinon.stub().returns(true),
+    };
```

Comment:
I'm not sure that it makes sense to mock these very simple functions, as opposed to passing an appropriate object created by the `annotation-fixtures` functions as the `annotation` prop. There are also several tests that repeat the initialization here (ie. making `isReply` and `isSaved` return false and true respectively).